src/OrchardCore.Modules/OrchardCore.Roles/ViewModels/PermissionGroupKey.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+
+namespace OrchardCore.Roles.ViewModels;
+
+public class PermissionGroupKey
+{
+    private readonly string _key;
+
+    public string Title { get; set; }
+
+    public string Source { get; set; }
```

Comment:
Where this is used in the class?!!

Reply:
The class is used in the controller for grouping

Reply:
I mean the `Source` property